types/nightwatch/index.d.ts

```diff
@@ -1468,8 +1847,8 @@ export interface NightwatchBrowser
 
 export interface NightwatchComponentTestingCommands {
     importScript(scriptPath: string, options: { scriptType: string; componentTyp: string }, callback: () => void): this;
-    mountReactComponent(componentPath: string, props?: string | (() => void), callback?: () => void): this;
-    mountVueComponent(componentPath: string, options?: any, callback?: () => void): this;
+    mountReactComponent(componentPath: string, props?: string | (() => void), callback?: () => void): Element;
+    mountVueComponent(componentPath: string, options?: any, callback?: () => void): Element;
```

Comment:
There is a command `mountComponent` in for `@nightwatch/react` plugin. Should we include that too?